02-social-links/style.css

```diff
@@ -0,0 +1,223 @@
+* {
+  margin: 0;
+  padding: 0;
+  /* box-sizing: content-box; */
+  font-family: "Inter", sans-serif;
+  font-optical-sizing: auto;
+  font-weight: 500;
+  font-style: normal;
+  font-variation-settings: "slnt" 0;
+  transition: all 0.2s;
+}
+article {
+  display: flex;
+}
```

Comment:
You made your `article` tag `flexbox`, but it only has 1 child `.container`. What's the purpose of setting it to flexbox?